src/main/java/lotto/service/PickLottoService.java

```diff
@@ -0,0 +1,37 @@
+package lotto.service;
+
+import static lotto.constants.LottoCondition.MIN_LOTTO_NUMBER;
+import static lotto.constants.LottoCondition.MAX_LOTTO_NUMBER;
+import static lotto.constants.LottoCondition.LOTTO_NUMBERS;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import lotto.model.Lotto;
+import lotto.model.PurchasedLottos;
+import lotto.model.TicketCount;
+
+public class PickLottoService {
+
+    private static final int INIT_IDX = 0;
+
+    public PickLottoService() {
+    }
+
+    public PurchasedLottos autoIssue(TicketCount ticketCount) {
+        List<Lotto> purchasedLottos = new ArrayList<>();
+        for (int nowIdx = INIT_IDX; nowIdx < ticketCount.get(); nowIdx++) {
+            purchasedLottos.add(nowIdx, new Lotto(getRandomNumbers()));
+        }
+        return new PurchasedLottos(purchasedLottos);
+    }
```

Comment:
이 부분 stream api 사용해서 리팩토링 하면 가독성이 더 높아질 것 같아요!
```suggestion
    public PurchasedLottos autoIssue(TicketCount ticketCount) {
    List<Lotto> purchasedLottos = IntStream.range(0, ticketCount.get())
            .mapToObj(i -> new Lotto(getRandomNumbers()))
            .collect(Collectors.toList());
    
    return new PurchasedLottos(purchasedLottos);
}
```
이런식으로요!

Reply:
Stream을 정말 잘 활용하시는 것 같아요!! 혹시 어떻게 공부하셨는지 알 수 있을까요? 👀

Reply:
프로그래머스 문제를 java로 문제 풀이 하면서 공부했습니다!

‘코딩 테스트 합격자 되기 java편’ 도서 참고했어요